src/System.Net.HttpListener/src/System/Net/HttpListenerPrefixCollection.cs

```diff
@@ -39,6 +39,11 @@ internal HttpListenerPrefixCollection(HttpListener listener)
         public void CopyTo(Array array, int offset)
         {
             _httpListener.CheckDisposed();
+            // Preserve compatability with full .NET Framework.
+            if (array == null)
+            {
+                throw new NullReferenceException();
```

Comment:
We don't need bug-for-bug compatibility.  It's a bug in desktop that it throws an NRE; we don't need to go out of our way to mimic that.